docs/howtoguides.rst

```diff
@@ -49,6 +49,16 @@ Fix vulnerabilities
 
    Fix vulnerabilities... <howtoguides/fix_index>
 
+Vulnerability commands
```

Comment:
I think it would be better to include this content under the previous heading (fix vulnerabilities). Do any of the pages in that section need to be updated with the new commands info as well?

Reply:
Even though they are related, I am still not sure if we should make a direct association here. But we can better discuss this for sure

Reply:
Discussed, resolving